### DISCUSSION TITLE
Laravel 11 support

### DISCUSSION THREAD

**Discussion by mityukov** [botman/botman#1397] [Q&A] [answered]
Hi, is there a support for Laravel 11 expected?

I'm asking, because I've found only one issue with this question: https://github.com/botman/botman/issues/1396

, and that issue was closed without any comments…

Is it in plans? If not, I'll probably fork, but I can wait if it's indeed coming :-)

**Comment by filippotoso** [comment, marked as answer]
https://github.com/botman/botman/commits/v2.8.3/

**Reply by mityukov** [reply]
Thank you! Can't see this version in Packagist, though. Composer update also doesn't see it.

**Reply by filippotoso** [reply]
Wait a couple of minutes and it should be downloadable using Composer.
